src/entity-manager.js

```
@@ -21,35 +21,36 @@ export class EntityManager {
   }
 
   /**
-   * Register an array of entity references.
+   * Register an array of entity classes.
    *
-   * @param {Entity[]|Entity} entities Array or object of entities.
+   * @param {function[]|function} Entity classes array or object of Entity constructors.
    *
    * @return {EntityManager} this
    * @chainable
    */
-  registerEntities(entities) {
-    for (let reference in entities) {
-      if (!entities.hasOwnProperty(reference)) {
+  registerEntities(EntityClasses) {
+    for (let property in EntityClasses) {
+      if (!EntityClasses.hasOwnProperty(property)
```

re instanceof: those are classes
re die hard: it that case it is supposed to fail silently. seems i forgot to mention the reason for this PR :smile: 
ah oh i sortof did. the reason is in the linked issue